spikeinterface/qualitymetrics/pca_metrics.py

```diff
@@ -28,8 +27,7 @@
 _default_params = dict(
     nearest_neighbor=dict(
         max_spikes=10000,
-        min_spikes=10,
-        n_neighbors=4,
+        n_neighbors=5,
```

Comment:
why this 4 to 5 by default ?

Reply:
It was 5 originally and we accidentally changed it